src/Sarif/Schemata/sarif-schema.json

```diff
@@ -367,6 +367,30 @@
       "required": [ "threadFlows" ]
     },
 
+    "configurationOverride": {
```

Comment:
>configurationOverride [](start = 5, length = 21)

OMG! So we had properties defined with `"$ref": "#/definitions/configurationOverride"`, but we never defined the type.

Reply:
nm, I see it was just moved from below.

---
In reply to: [272240688](https://github.com/Microsoft/sarif-sdk/pull/1400#discussion_r272240688) [](ancestors = 272240688)